include/revng/Pypeline/Helpers/Python/AnalysisRunnerImpl.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "nanobind/nanobind.h"
+
+#include "revng/Pypeline/Helpers/AnalysisRunner.h"
+#include "revng/Pypeline/Helpers/Python/RunnerInfo.h"
+#include "revng/Pypeline/Helpers/Python/Utils.h"
+#include "revng/Pypeline/Model.h"
+
+namespace revng::pypeline::helpers::python {
+
+template<typename T>
+struct AnalysisRunnerImpl {
```

Comment:
Why do we need a `struct` here? Seem like a freestanding function to me.

Reply:
Converted this and the pipe counterpart to functions